src/common/logger.go

```diff
@@ -0,0 +1,22 @@
+package common
+
+import (
+	"log/slog"
+	"os"
+)
+
+// Default logger for the entire system (Any call from log will immediately be written with a JSON handler)
+// TODO: Only format it as JSON for cgroups but any other systems calling log.Printf should just get a regular log output
+var defaultSystemLogger = slog.New(slog.NewJSONHandler(os.Stdout, nil))
```

Comment:
The config should have an option about whether to use JSON our Text formatting of output.

Reply:
I like how https://github.com/open-lambda/open-lambda/pull/203 was handling the config.  Can we do something similar?

Reply:
We need to handle different logging levels for different subsystems.  How will that work out?  It seems like you could create different handlers (since you specify the level when you create the handler), but will we have multiple locks and jumbled output then?